sormas-backend/src/main/java/de/symeda/sormas/backend/systemevent/SystemEventFacadeEjb.java

```diff
@@ -56,21 +58,20 @@ public SystemEventDto getLatestSuccessByType(SystemEventType type) {
 		CriteriaQuery<SystemEvent> cq = cb.createQuery(SystemEvent.class);
 		Root<SystemEvent> systemEventRoot = cq.from(SystemEvent.class);
 
-		cq.where(cb.equal(systemEventRoot.get(SystemEvent.STATUS), SystemEventStatus.SUCCESS));
+		final Predicate equalSystemEventStatus = cb.equal(systemEventRoot.get(SystemEvent.STATUS), SystemEventStatus.SUCCESS);
```

Comment:
I would not assign these as method variables, but directly inline them into the WHERE method.